scripts/cleanup_clinvar.py

```diff
@@ -0,0 +1,96 @@
+"""
+A script to create a suitable ClinVar reference vcf that:
+ - Removes contigs that are not in your fasta
+ - Uses chr contig nomenclature
+ - Uses a variants_summary file to break up N alts to usage ACGT alleles
+"""
+
+import pysam
+import argparse
+import json
+import gzip
+import pdb
+
+
+def split_records(record, nt_list, out):
+    for nt in nt_list:
+        record.alts = (nt,)
+        out.write(record)
+
+
+def update_contig_header(contig_list, header):
+    for contig in contig_list:
+        header.contigs.add(contig)
+
+
+def create_mod_vcf(output_path, input_path, threads, update_dict, non_canon_dict, var_sum_path, update_path):
+    input_vcf = pysam.VariantFile(input_path, 'r', threads=threads)
+    # add new contigs to header
+    update_contig_header(update_dict['renamed_contigs'], input_vcf.header)
+    # add program command for pizzaz
+    input_vcf.header.add_meta("cleanup_clinvar.py", "--input_vcf {} --variant_summary {} --update_json {} --output_filename {} --threads {}".format(input_path, var_sum_path, update_path, output_path, threads))
+
+    output = pysam.VariantFile(output_path, 'w', header=input_vcf.header)
+    cur = ""
+    rename = ""
+    for record in input_vcf.fetch():
+        try:
+            if record.contig in update_dict['old_contigs']:
+                if record.contig != cur:
+                    cur = record.contig
+                    if cur in update_dict['old_contigs']:
+                        rename = update_dict['renamed_contigs'][update_dict['old_contigs'].index(cur)]
+                record.contig = rename
+                # Use variant id in non-canon dict to update alts
+                if record.id in non_canon_dict:
+                    update_alts = update_dict['iupac'][non_canon_dict[record.id]]
+                    split_records(record, update_alts, output)
```

Comment:
so I'm unsure if this is needed. BCFtools doesn't need the records to be decomposed. According to their documentation:
> When multiple ALT alleles are present in the annotation file (given as comma-separated list of alleles), at least one must match one of the alleles in the corresponding VCF record. Similarly, at least one alternate allele from a multi-allelic VCF record must be present in the annotation file.